pyhanko/sign/signers/pdf_signer.py

```diff
@@ -2141,13 +2164,26 @@ def prepare_tbs_document(
             timestamp=system_time,
             text_params=appearance_text_params,
         )
+
+        # According to ISO, if /SubFilter is ETSI_RFC3161
+        # /Prop_AuthTime shall not be used
+        if self.subfilter == SigSeedSubFilter.ETSI_RFC3161:
```

Comment:
I'd rather throw an error here if `prop_auth_XYZ` is set, instead of silently discarding the values of the property.

That said, you mostly don't have to worry about this since in 99.99% of cases RFC 3161 document timestamps are generated by pyHanko in the background without much opportunity for user input. In particular, there's no way to get to this point with signer info parameters from the CLI.

So I'd be OK with _not_ having this safeguard too---I think it's fairly safe to assume that a caller passing in this `SubFilter` value knows what they're doing.

Reply:
Ah, apparently pyHanko even forbids using this subfilter value in regular signatures (see failing tests). In that case, IMHO it's cleaner to remove the conditional and the associated tests.

Reply:
I partially agree with you. I also think people should know what they're doing, but in many cases they don't. Just thought pyHanko could make this verification in order to prevent users to generate invalid documents. However, as pyHanko forbids this subfilter, it's not a problem to be treated now.